pkg/slo-controller/metrics/node_resource.go

```diff
@@ -0,0 +1,60 @@
+/*
+Copyright 2022 The Koordinator Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package metrics
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	corev1 "k8s.io/api/core/v1"
+
+	"github.com/koordinator-sh/koordinator/pkg/util/metrics"
+)
+
+func init() {
+	MustRegister(NodeResourceCollectors...)
+}
+
+var (
+	NodeResourceRunPluginStatus = metrics.NewGCCounterVec("node_resource_run_plugin_status", prometheus.NewCounterVec(prometheus.CounterOpts{
```

Comment:
only metrics related to node/pod object should use GC metrics (has NodeKey/PodKey).

Reply:
fixed